### PR TITLE
Copyright update to reflect IP transfer from Salvatore to Redis

### LINKED ISSUE #544: [LEGAL] Please remove my copyright notice from the source code
Dear friends,

sorry to bother you with this request, but the copyright setups I reached in the past with Redis Ltd demands some change.
Years ago, I sold the copyright of the Redis source code, the brand and the logo to Redis Ltd. As you know, selling the copyright of a BSD-licensed piece fo code does not lower the freedom other parties have with the source code, as long as the BSD license is retained, but it changes the legal copyright owner, so after a given date I'm no longer the copyright holder.

It is likely possible to fork the Redis source code and retain the copyright if you pick a commit that is before my agreement with Redis Ltd, but this dates to many years ago, so given that you fork was performed with the latest sources, both you and I are not complying with the terms. Note that this is **entirely my fault** as when I provided Redis Ltd with the copyright of Redis, I needed to switch the copyright in the source code, but I didn't. Please let's fix this together.

Note that, thanks to the BSD license of the code you used before the fork, other than deleting my copyright notice you don't need to do any other change (but please ask your lawyer as I'm not one). Otherwise the code is usable for whatever goal the BSD license permits.

Good hacking to any Redis tree out there, whatever the license is :D

Cheers,
Salvatore

### PR DESCRIPTION
**Pull request by K-Jo**
Update references of copyright being assigned to Salvatore when it was
transferred to Redis Ltd. as per https://github.com/valkey-io/valkey/issues/544.

### CODE REVIEW

**Comment by madolson** [comment]
Sorry for the delay, we were working on RC1. Will go through it today to double check the updates and then merge it.

File: deps/hiredis/COPYING [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2011, Salvatore Sanfilippo <antirez at gmail dot com>
+Copyright (c) 2009-2011, Redis Ltd.
```
Comment (madolson) [inline deps/hiredis/COPYING:1]: FYI, this code was also vendored in hiredis, so will also be present in libvalkey https://github.com/valkey-io/libvalkey/blob/main/libvalkey/COPYING. Not sure if you also want to update that.

**Review by madolson — DISMISSED** [review]
LGTM

**Comment by madolson** [comment]
You also seem to have missed a couple, like https://github.com/valkey-io/valkey/blob/0fc43edc6cf388854bc824f192647cc59f7019e7/tests/support/valkey.tcl#L2

EDIT: Now noticing that thse had (C) instead of (c), might be a regex problem.

**Review by madolson — COMMENTED** [review]
I noticed you missed one reference to the copyright, so I took a deeper look and found a few more.

Just to list out all the Salvatore copyright references still present:
1. utils/*: I assume you don't care about these
2. COPYING: I assumed you omitted this one intentionally originally since it's sort of the main one that needs to be retained, but maybe it was missed. 
3. deps/linenoise/*: Seems missed, but maybe intentional.
4. deps/lua/src/lua_cmsgpack: A bunch of references to Salvatore trademark in the msgpack code.
5. src/(crc16.c,listpack.c,psqsort.c,setproctitle.c): These had a capital C in the copyright, so I'm guessing your regex just missed these.
6. tests/support/(cluster.tcl,valkey.tcl): More capital Cs, so probably also missed.

**Comment by madolson** [comment]
@K-Jo Gentle ping if you still intend to update this.

**Comment by K-Jo** [comment]
@madolson Thanks for your feedback. I pushed another commit to fix the issues you mentioned.

**Review by madolson — APPROVED** [review]

**Pull request merged**